centreon/www/class/config-generate/AdditionalConnectorVmWareV6.class.php

```diff
@@ -48,7 +50,7 @@ public function __construct(
      *
      * @param int $pollerId
      *
-     * @throws \Exception|AssertionFailedException
+     * @throws \Exception|AssertionFailedException|\RuntimeException
```

Comment:
RuntimeException extends Exception
```suggestion
     * @throws \Exception|AssertionFailedException
```